{"stackTrace": [["/var/task/get_test.py", 44, "lambda_handler", "raise Exception('\"NFP-Error-Code:400059\"')"]], "errorType": "Exception", "errorMessage": "\"NFP-Error-Code:400059\""}